test/services/licences/determine-licence-has-return-versions.service.test.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../../support/database.js')
+const ReturnVersionHelper = require('../../support/helpers/return-version.helper.js')
+
+// Thing under test
+const FetchLicenceHasRequirementsService =
+  require('../../../app/services/licences/determine-licence-has-return-versions.service.js')
+
+describe('Fetch Licence Has Requirements service', () => {
+  let returnVersion
+
+  beforeEach(async () => {
+    await DatabaseSupport.clean()
+  })
+
+  describe('when the licence has return versions data', () => {
+    beforeEach(async () => {
+      returnVersion = await ReturnVersionHelper.add()
+    })
+
+    it('returns true a licence has any return versions', async () => {
+      const result = await FetchLicenceHasRequirementsService.go(returnVersion.licenceId)
+
+      expect(result).to.be.true()
+    })
+
+    it('returns false no return versions for licence', async () => {
+      const result = await FetchLicenceHasRequirementsService.go('ed3b9b1a-94e0-480c-8ad6-60e05f5fa9f4')
+
+      expect(result).to.be.false()
+    })
+  })
```

Comment:
The 'returns false' test needs to be outside of the describe block because it currently reads as; "when the licence has return versions data, it returns false no return versions for licence".

I'd also say that the data setup makes things a little confusing. I'd suggest a slight tweak might make it clearer.

```suggestion
  const licenceId = 'e004c0c9-0316-42fc-a6e3-5ae9a271b3c6'

  beforeEach(async () => {
    await DatabaseSupport.clean()
  })

  describe('when the licence has return versions', () => {
    beforeEach(async () => {
      await ReturnVersionHelper.add({ licenceId })
    })

    it('returns true', async () => {
      const result = await FetchLicenceHasRequirementsService.go(licenceId)

      expect(result).to.be.true()
    })
  })

  describe('when the licence does not have return versions', () => {
    it('returns false', async () => {
      const result = await FetchLicenceHasRequirementsService.go('ed3b9b1a-94e0-480c-8ad6-60e05f5fa9f4')

      expect(result).to.be.false()
    })
  })
```